### DISCUSSION TITLE
reveal.js : how to set  specific figure size with R ?

### DISCUSSION THREAD

**Discussion by wilcar** [quarto-dev/quarto-cli#833] [Q&A] [answered]
I am using mermaid within DiagrammeR package to create an ER diagram  but my output is very small. How can I set the output size in revealj presentation ?

    ## slide
```{=html}
<script src="https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min.js"></script>
```
```{=html}
<script>mermaid.initialize({startOnLoad:true});</script>
```
 ```{r}
 library(DiagrammeR)
mermaid("
erDiagram
FROM ||--|{ TO : Flow
")
```

**Comment by cderv** [comment]
Hello ! 

Quarto has now built in support for mermaid graph. You may want to try that: https://quarto.org/docs/authoring/diagrams.html

**Reply by cscheid** [reply]
(@cderv full sizing support for `mermaid` and `dot` hasn't landed yet) I'll post here when we have it!

**Comment by cscheid** [comment, marked as answer]
This is fresh off the latest commit. This markdown:

    ---
    format: revealjs
    ---
    
    ## slide
    
    ```{mermaid}
    %%| fig-height: 6
    erDiagram
      FROM ||--|{ TO : Flow
    ```

produces the following result:

<img width="764" alt="image" src="https://user-images.githubusercontent.com/285675/168392920-7dff3f37-d912-4264-b1f6-4d457f5fe5f6.png">
